light-base/examples/basic.rs

```diff
@@ -81,32 +79,70 @@ fn main() {
             user_data: (),
         })
         .unwrap();
-
-    // The chain is now properly initialized.
+    // Connection to Polkadot is now properly initialized.
 
     // `json_rpc_responses` can only be `None` if we had passed `json_rpc: Disabled` in the
     // configuration.
-    let mut json_rpc_responses = json_rpc_responses.unwrap();
+    let mut polkadot_json_rpc_responses = polkadot_connection.json_rpc_responses.unwrap();
 
-    // Send a JSON-RPC request to the chain.
+    // Ask the client to connect to Assethub.
+    let assethub_connection = client
+        .add_chain(smoldot_light::AddChainConfig {
+            specification: include_str!("../../demo-chain-specs/asset-hub-polkadot.json"),
+            json_rpc: smoldot_light::AddChainConfigJsonRpc::Enabled {
+                max_pending_requests: NonZeroU32::new(128).unwrap(),
+                max_subscriptions: 1024,
+            },
+            // Providing the allocated identifier for Polkadot.
```

Comment:
This needs a better explanation.

```suggestion
            // The chain specification of the asset hub parachain mentions that the identifier
            // of its relay chain is `polkadot`. Because the `Client` might contain multiple different
            // chains whose identifier is `polkadot`, we need to provide a list of all the chains
            // that the `Client` should consider when searching for the relay chain. The `add_chain`
            // function returns an error if there is no match or if there are multiple matches when
            // searching for an appropriate relay chain within this list.
```